Boost 1.61 on Mac

$ cmake ..
-- DeNovoGear version is 1.1.1-207-g72d43d2
-- Boost version: 1.61.0
-- Found the following Boost libraries:
--   program_options
--   filesystem
--   system
--   unit_test_framework
CMake Warning at Modules/LibFindMacros.cmake:191 (message):
  The find module for HTSLIB does not provide version information, so we'll
  just assume that it is OK.  Please fix the module or remove package version
  requirements to get rid of this warning.
Call Stack (most recent call first):
  Modules/FindHTSLIB.cmake:15 (libfind_process)
  Modules/ExternalDep.cmake:299 (FIND_PACKAGE)
  CMakeLists.txt:209 (INCLUDE)

-- Found HTSLIB 
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/nate/github/denovogear/build

$ make
[  3%] Building CXX object src/lib/CMakeFiles/libdng.dir/call.cc.o
In file included from /Users/nate/github/denovogear/src/lib/call.cc:40:
In file included from /Users/nate/github/denovogear/src/lib/../include/dng/pileup.h:24:
/Users/nate/github/denovogear/src/lib/../include/dng/pool.h:86:23: error: 'safemode_or_autounlink' is a private member of
      'boost::intrusive::list_impl<boost::intrusive::bhtraits<dng::pileup::Node, boost::intrusive::list_node_traits<void *>,
      boost::intrusive::link_mode_type::normal_link, boost::intrusive::dft_tag, 1>, unsigned long, true, void>'
        if(list_type::safemode_or_autounlink) {
                      ^
/Users/nate/github/denovogear/src/lib/../include/dng/pileup.h:170:5: note: in instantiation of member function
      'dng::IntrusivePooldng::pileup::Node::~IntrusivePool' requested here
    BamPileup() { }
    ^
/usr/local/include/boost/intrusive/list.hpp:123:22: note: declared private here
   static const bool safemode_or_autounlink = is_safe_autounlink<value_traits::link_mode>::value;
                     ^
1 error generated.
make[2]: **\* [src/lib/CMakeFiles/libdng.dir/call.cc.o] Error 1
make[1]: **\* [src/lib/CMakeFiles/libdng.dir/all] Error 2
make: **\* [all] Error 2

--- a/src/include/dng/pool.h
+++ b/src/include/dng/pool.h
@@ -83,7 +83,7 @@ public:
             return;
         }
         // clear inactive as needed
-        if(list_type::safemode_or_autounlink) {
+	if(boost::intrusive::is_safe_autounlink<list_type::value_traits::link_mode>::value) {
             inactive_.clear();
         }
         // enumerate over allocated blocks

